doc/HOWTO_RELEASE.rst.txt

```diff
@@ -337,6 +326,22 @@ strides are used in the development branch to smoke out problems. The
 `patch <https://github.com/numpy/numpy/pull/5996>`_ should be updated if
 necessary and applied to the release branch to rationalize the strides.
 
+Build and archive documentation
+-------------------------------
+Do::
+
+    cd doc/
+    make dist
+
+to check that the documentation is in a buildable state. Then, after tagging,
+create an archive of the documentation in the numpy/doc repo::
+
+	make merge-doc
+	# Read output comments and edit doc/dist/merge/index.html as needed
+	git -C dist/merge commit -a"Add documentation for <version>"
+	# Push to numpy/doc repo
+	git -C push
```

Comment:
what does `-C` mean (hard to find in docs, and very uncommon)?

Reply:
I think this is incomplete, and should be `cd dist/merge && git push` (which is clearer)

Reply:
"-C" is supported in [git](https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt) like in tar, mercurial, and perhaps other tools. Added a comment before.